File: go/wam/2862.go
const WamPsIdUpdate = 2862 // channel: regular

type PsIdUpdateEvent struct {
	PsIdKey int64 `wam:"ps_id_key"` // field 1
	PsIdAction PSIDACTION `wam:"ps_id_action"` // field 2
	PsIdRotationFrequence int64 `wam:"ps_id_rotation_frequence"` // field 3
	IsFromWamsys bool `wam:"is_from_wamsys"` // field 4
}

File: rust/wam/2862.rs
pub const WAM_PS_ID_UPDATE: u32 = 2862;

#[derive(Debug, Default)]
pub struct PsIdUpdateEvent {
    /// field 1, wire `ps_id_key`
    pub ps_id_key: Option<i64>,
    /// field 2, wire `ps_id_action`
    pub ps_id_action: Option<PSIDACTION>,
    /// field 3, wire `ps_id_rotation_frequence`
    pub ps_id_rotation_frequence: Option<i64>,
    /// field 4, wire `is_from_wamsys`
    pub is_from_wamsys: Option<bool>,
}